File: help/eng_gb/exercicios da ficha_11/Ficha11_005/Ficha11_005.php
<?php
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    line() ;
    echo  "Numbers between 1 and 5\n" ;
    line() ;
    for( $i = 1; $i < 6; $i = $i + 1 ) {
        echo $i  .  "\n" ;
    }
    line() ;
}

//::::::::::::::::::::::::::::::::::::::::::::::::::
//User Defined Functions Code
//::::::::::::::::::::::::::::::::::::::::::::::::::
function line() {
    for( $i = 0; $i < 20; $i = $i + 1 ) {
        echo  "*" ;
    }
    echo  "\n" ;
}

main();

//PHP 5.3 +  - Automatic translated by Algorithmi 22.05
//(c) Luís Lopes 	quarta, 2022 out 12 20:22

?>

File: help/eng_gb/exercicios da ficha_11/Ficha11_005/Ficha11_005.c
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <stdio.h>
#include <string.h>
#include <math.h>
//User Defined Functions
void line(  );

//Main Function
int main() {
    line() ;
    printf("Numbers between 1 and 5\n");
    line() ;
    int i = 0 ;
    for( int i = 1; i < 6; i = i + 1 ) {
        printf("%d\n",i );
    }
    line() ;
    return 0;
}

//::::::::::::::::::::::::::::::::::::::::::::::::::
//User Defined Functions Code
//::::::::::::::::::::::::::::::::::::::::::::::::::
void line(  ){
    int i = 0 ;
    for( int i = 0; i < 20; i = i + 1 ) {
        printf("*");
    }
    printf("\n");
}


//C Language c11  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 12 20:22
